Automattic/wc-calypso-bridge
# Add WC Calypsoify support for bundled extensions

The PR ended up being small, but I did go through and test/confirm the following extensions are Calypsoified and show up in the correct menu sections. A good deal of them don't need extra code because they hook into the payment settings, shipping settings, integration tabs, or the customizer. All the extensions I tested as a part of this PR make up the bundled package.

Extensions Tested:

* Facebook
* MailChimp
* Storefront
* Product Add-Ons
* Storefront PowerPack
* TaxJar
* Stripe
* PayPal Express Checkout
* eWay
* Square
* Klarna
* PayFast
* USPS
* Canada Post
* WooCommerce Services
* Royal Mail
* UPS
* Australia Post

To Test:
* Install some (or all) of these extensions and verify they show up in the right section.
